internal/buildscripts/packaging/installer/install.ps1

```diff
@@ -64,6 +64,21 @@
     (OPTIONAL) Whether to install and configure fluentd to forward log events to the collector (default: $true)
     .EXAMPLE
     .\install.ps1 -access_token "ACCESSTOKEN" -with_fluentd $false
+.PARAMETER with_dotnet_instrumentation
+    (OPTIONAL) Whether to install and configure .NET tracing to forward .NET application traces to the local collector (default: $false)
+    .EXAMPLE
+    .\install.ps1 -access_token "ACCESSTOKEN" -with_dotnet_instrumentation $true
+.PARAMETER instrumentation_exclude_processes
+    (OPTIONAL) A semicolon-delimited list of process names to be excluded from auto-instrumentation (default: Powershell.exe;dotnet.exe)
+    .\install.ps1 -access_token "ACCESSTOKEN" -with_dotnet_instrumentation $true -instrumentation_exclude_processes "Powershell.exe;dotnet.exe;myprogram.exe"
+.PARAMETER signalfx_service_name
```

Comment:
Keeping the transition to OTel .NET auto instrumentation in mind once it is stable in the future (alpha today) and we want to move away from signalfx references, I suggest renaming the service name and env parameters to avoid these changes in the future @pmcollins  - wdyt?

Reply:
Should they be renamed as follows?

`signalfx_service_name` -> `service_name`
`signalfx_env` -> `profiling_env`

I'm not sure we need to do this though because we generate the arguments and download the script that processes those arguments on demand. When we move over to OTel, we can just start using OTel naming without worrying about compatibility.

Reply:
Got it, thanks Pablo. Since these are parameters exposed as options for the installer, it would be best to rename the environment option upfront (because of the feedback we have received about naming in the past) while the SIGNALFX_ENV env variable is set underneath. I would suggest the following:
```
signalfx_env -> deployment_env
```

Reply:
The service name override option is now gone. The wizard should set the sfx env.

Reply:
Sorry, Aunsh, I didn't see your message when I responded. I'll change sfx env to deployment env shortly.

Reply:
No worries, I added a comment around the same time the changes we pushed I believe. Thanks!

Reply:
Ok, this is done.